django/core/cache/backends/base.py

```diff
@@ -388,6 +388,12 @@ def close(self, **kwargs):
     async def aclose(self, **kwargs):
         pass
 
+    def check(self, **kwargs):
+        raise NotImplementedError(
+            "subclasses may provide a check() method to verify the finder is "
+            "configured correctly."
+        )
```

Comment:
The base implementation should return an empty list instead of raising exceptions.